### ISSUE TITLE
Existing app.route()s triggering 404s in logs

### ISSUE THREAD

**Issue by adamhgriffith-uofu** [slateci/slate-portal#193]
On dev (`1.0.38-pre.20221212-170114`):
```
2022-12-12 10:29:19.073 MST
35.191.15.166 - - [12/Dec/2022 17:29:19] "GET /public-groups-xhr HTTP/1.1" 200 -
2022-12-12 10:29:19.074 MST
35.191.15.166 - - [12/Dec/2022 17:29:19] "[32mGET /cluster-status-xhr/umich-prod HTTP/1.1[0m" 302 -
2022-12-12 10:29:19.078 MST
35.191.15.172 - - [12/Dec/2022 17:29:19] "[32mGET /cluster-status-xhr/uutah-prod HTTP/1.1[0m" 302 -
2022-12-12 10:29:19.082 MST
35.191.15.160 - - [12/Dec/2022 17:29:19] "GET /users_instances_ajax HTTP/1.1" 200 -
2022-12-12 10:29:19.086 MST
35.191.15.163 - - [12/Dec/2022 17:29:19] "[32mGET /cluster-status-xhr/uchicago-prod HTTP/1.1[0m" 302 -
2022-12-12 10:29:19.117 MST
35.191.15.160 - - [12/Dec/2022 17:29:19] "[32mGET /login?next=http%3A%2F%2Fportal.dev.slateci.io%2Fcluster-status-xhr%2Fumich-prod HTTP/1.1[0m" 302 -
2022-12-12 10:29:19.127 MST
35.191.15.172 - - [12/Dec/2022 17:29:19] "[32mGET /login?next=http%3A%2F%2Fportal.dev.slateci.io%2Fcluster-status-xhr%2Fuutah-prod HTTP/1.1[0m" 302 -
2022-12-12 10:29:19.129 MST
35.191.15.163 - - [12/Dec/2022 17:29:19] "[32mGET /login?next=http%3A%2F%2Fportal.dev.slateci.io%2Fcluster-status-xhr%2Fuchicago-prod HTTP/1.1[0m" 302 -
2022-12-12 10:29:19.341 MST
35.191.15.166 - - [12/Dec/2022 17:29:19] "[32mGET /authcallback?next=http%3A%2F%2Fportal.dev.slateci.io%2Fcluster-status-xhr%2Fuutah-prod HTTP/1.1[0m" 302 -
2022-12-12 10:29:19.433 MST
35.191.15.163 - - [12/Dec/2022 17:29:19] "GET /applications_ajax HTTP/1.1" 200 -
2022-12-12 10:29:19.650 MST
35.191.15.169 - - [12/Dec/2022 17:29:19] "[32mGET /authcallback?next=http%3A%2F%2Fportal.dev.slateci.io%2Fcluster-status-xhr%2Fumich-prod HTTP/1.1[0m" 302 -
2022-12-12 10:29:19.652 MST
35.191.15.172 - - [12/Dec/2022 17:29:19] "[32mGET /authcallback?next=http%3A%2F%2Fportal.dev.slateci.io%2Fcluster-status-xhr%2Fuchicago-prod HTTP/1.1[0m" 302 -
2022-12-12 10:29:22.900 MST
/usr/local/lib/python3.9/site-packages/flask/app.py:1774: UserWarning: Current server name '10.44.2.242:5050' doesn't match configured server name 'portal.dev.slateci.io'
2022-12-12 10:29:22.901 MST
return self.url_map.bind_to_environ(
2022-12-12 10:29:22.901 MST
[2022-12-12 17:29:22,899] ERROR in views_error_handling: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.. URL: /healthz
2022-12-12 10:29:22.901 MST
10.44.2.1 - - [12/Dec/2022 17:29:22] "GET /healthz HTTP/1.1" 200 -
```

**Comment by adamhgriffith-uofu** [comment]
Some things to try: https://stackoverflow.com/questions/24437248/unexplainable-flask-404-errors

**Comment by adamhgriffith-uofu** [comment]
I was able to quiet the logs for the underlying `werkzeug` library. `werkzeug` does not like being in a container with `app.run(host='0.0.0.0', ...)` and `SERVER_NAME=portal.dev.slateci.io`. In the linked docs above they talk about ways to work around this but I did not have much success.

So apart from wrapping and serving this application from something like https://github.com/tiangolo/uwsgi-nginx-flask-docker we might have to re-write some code. Right now I don't think it is worth it.
